hooks/generic-webhook/values.yaml

```diff
@@ -35,8 +35,14 @@ hook:
   # hook.tolerations -- Optional tolerations settings that control how the hook job is scheduled (see: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
   tolerations: []
 
-  # hook.authentication -- Optional basic authentication credentials
+  # hook.authentication -- Optional basic authentication credentials or apikey
   authentication:
+    apikey:
+      # -- Link a pre-existing generic secret with `usernameKey` and `passwordKey` key / value pairs
+      userSecret: generic-webhook-credentials
+      # -- Customize header name as per your needs
+      key: X-api-key
+      secret: 26ea529e517748baa6d87ebfe5781475
```

Comment:
This also doesn't work as intended.
The key / secret should not be specified here using the helm values but should be put in a pre-existing secret.
So the user creates the secret beforehand and then only links to the secret from the helm values. I've noted how remove them in the templates above.

Reply:
Sorry @J12934 I don't understand.

Reply:
@J12934, check my corrections please?